app/templates/components/component-tree-item.hbs

```diff
@@ -21,10 +20,81 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag
+      {{if
+        (and (not @item.isCurlyInvocation) @item.isComponent)
+        "component-tree-item__bracket"
+      }}
+      {{if
+        (and (not @item.isCurlyInvocation) (not @item.hasChildren) @item.isComponent)
+        "component-tree-item__bracket--self-closing"
+      }}
+      {{if
+        (and @item.isCurlyInvocation @item.isComponent)
+        "component-tree-item-classic__bracket"
+      }}"
```

Comment:
```hbs
{{if @item.isComponent
  (if @item.isCurlyInvocation
    "component-tree-item-classic__bracket"
    (if @item.hasChildren
      "component-tree-item__bracket--self-closing"
      "component-tree-item__bracket"
    )
  )
}}
```

(you can indent it however the linter wants it, but we should group the conditions)